mmv1/third_party/terraform/tests/resource_dataflow_job_test.go.erb

```diff
@@ -46,6 +46,12 @@ func TestAccDataflowJob_basic(t *testing.T) {
 					testAccDataflowJobExists(t, "google_dataflow_job.big_data"),
 				),
 			},
+			{
+				ResourceName:            "google_dataflow_job.big_data",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"on_delete", "parameters", "skip_wait_on_job_termination", "zone"}
```

Comment:
```suggestion
				ImportStateVerifyIgnore: []string{"on_delete", "parameters", "skip_wait_on_job_termination", "zone"},
```
I think you'll need a comma after each of these. Thanks!

Reply:
Solved